MapboxNavigationTests/Fixtures/Fixture.swift

```diff
@@ -62,6 +62,19 @@ internal class Fixture {
         let jsonRoute = (response["routes"] as! [AnyObject]).first as! [String : Any]
         return Route(json: jsonRoute, waypoints: waypoints, options: RouteOptions(waypoints: waypoints))
     }
+    
+    // Returns `Route` objects from a match response
+    class func routesFromMatches(at filePath: String) -> [Route]? {
+        let path = Bundle(for: Fixture.self).path(forResource: filePath, ofType: "json")
+        let url = URL(fileURLWithPath: path!)
+        let data = try! Data(contentsOf: url)
+        let json = try! JSONSerialization.jsonObject(with: data, options: []) as! JSONDictionary
+        let tracepoints = json["tracepoints"] as! [Any]
+        let coordinates = (0...tracepoints.count-1).map { _ in CLLocationCoordinate2D(latitude: 0, longitude: 0) }
+        let options = MatchOptions(coordinates: coordinates, profileIdentifier: .automobile)
+        let response = options.response(containingRoutesFrom: json)
```

Comment:
@nitaliano noticed that the test target no longer builds on Xcode 10 locally because of the use of this method and `@testable` imports that require the MapboxDirections module to be built for testing.